redis-util/src/main/java/org/sunbird/redis/RedisCacheUtil.java

```diff
@@ -0,0 +1,100 @@
+package org.sunbird.redis;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import org.redisson.api.RMap;
+import org.redisson.api.RedissonClient;
+import org.sunbird.cache.inf.CacheUtil;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+
+import com.typesafe.config.ConfigFactory;
+import com.typesafe.config.ConfigValue;
+
+public class RedisCacheUtil implements CacheUtil {
+  protected static final String REDIS_CONFIG_FILE = "cache.conf";
+  private static final String CACHE_MAP_LIST = "cache.mapNames";
+  private Map<String, String> properties = readConfig();
+  private String mapNameListStr = properties.get(CACHE_MAP_LIST);
+  private String[] mapNameList = mapNameListStr.split(",");
+  private  RedissonClient client = null;
+
+  public RedisCacheUtil() {
+    client =  RedisConnectionManager.getClient();
+    
+  }
+
+  @Override
+  public Map<String, String> readConfig() {
```

Comment:
readConfig implementation can be part of interface as it could be reusable across implementations.